experimental/auth-logic/endorsement-release/input_policy_examples/github_actions_policy.auth_logic

```diff
@@ -0,0 +1,11 @@
+"GithubActionsPolicy" says {
```

Comment:
Is this file auto-generated? I have seen repos that have a `generated` folder for auto-generated files. I think that is useful conventions that we could use too.